### PR TITLE
History tutorial update

### PR DESCRIPTION
**Pull request by nekrut**
The idea is to:

1. Update this to 24.0
2. Introduce the concept of individual datasets and collections
3. Explain how to trace dataset inheritance
4. Explain multi-history view

### CODE REVIEW

File: topics/galaxy-interface/images/history.svg
Comment (hexylena) [inline topics/galaxy-interface/images/history.svg:1]: could you please crop these a bit better

File: topics/galaxy-interface/tutorials/history/tutorial.md [hunk 1]
```diff
@@ -62,31 +62,44 @@ benefit that they can work on many histories and switch between them.
...
+- The {% icon switch-histories %} "**Switch to history**" will open a dialog letting you easily swap to any of your other histories.
+- The {% icon galaxy-history-options %} "**History options**" (formerly the {% icon galaxy-gear %} "Gear menu") gives you access to advanced options to work with your history.
 
-- The {% icon new-history %} 'create new history' button will create an empty history.
-- The {% icon switch-histories %} 'history quick switcher' will open a dialog letting you easily swap to any of your other histories.
-- The {% icon galaxy-history-options %} 'history menu' (formerly the {% icon galaxy-gear %} "Gear menu") gives you access to advanced options to work with your history.
+![Zoom of the history panel showing the word history, three buttons described above this image, the search datasets field, and the history name with edit control.](../../images/current-history-buttons.png "Global history controls are located above the history panel.")
 
-## History Information
+### {% icon new-history %} "**Create new history**"
```
Comment (hexylena) [inline topics/galaxy-interface/tutorials/history/tutorial.md:75]: This is a nice idea, that'll make the icons more visible.

**Review by hexylena — DISMISSED** [review]
This looks like an improvement, thank you @nekrut. I look forward to merging it as soon as the image is cropped

File: topics/galaxy-interface/images/history.svg
Comment (nekrut) [reply topics/galaxy-interface/images/history.svg:1]: this is not done - it is still a day's worth of work there

Comment (hexylena) [reply topics/galaxy-interface/images/history.svg:1]: Then I will mark it as draft, please mark it as ready for review when you're done :)

File: topics/galaxy-interface/tutorials/history/tutorial.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ The lower part of the history header contains a number of buttons:
 
 ![Buttons for figuring out history size, selecting storage location, and controlling views](../../../../shared/images/history_size_storage_views.png "Using these buttons one can obtain an information about history's disk footpring, change storage options, and control dataset views.")
 
-> <warning-title>Some icons are instance-specific</warning-title>
+> <warning-title>Some buttons are instance-specific</warning-title>
 > The list of buttons shown above may vary depending on which Galaxy instance you are using. For example, at the tim eof writing the **Preferred storage** {% icon galaxy-history-storage-choice %} button is only available on https://usegalaxy.org.
```
Comment (hexylena) [inline topics/galaxy-interface/tutorials/history/tutorial.md:104]:
```suggestion
> The list of buttons shown above may vary depending on which Galaxy instance you are using. For example, at the time of writing the **Preferred storage** {% icon galaxy-history-storage-choice %} button is only available on https://usegalaxy.org.
```

File: faqs/galaxy/histories_undelete.md [hunk 3]
```diff
@@ -4,10 +4,14 @@ description: Undelete your deleted histories
...
-* On **Status** click **Deleted**
-* Select the history you want to undelete using the **checkbox** on the left side
-* Click **Undelete** button below the deleted histories
+
+If you have not purged a history and have only deleted it, it is possible to 'undelete' it and reverse or undo the deletion. To do so:
+
+1. Select "Histories" from the activity bar of the left
+2. Toggle "Advanced search"
+3. Click "Deleted"
+4. Click on the title of the history you want to un-delete, and un-delete it.
```
Comment (bgruening) [inline faqs/galaxy/histories_undelete.md]: this sounds a bit bumpy

File: Gemfile.lock
Comment (bgruening) [inline Gemfile.lock:1]: Anton, can you please revert those changes here?

Comment (nekrut) [reply Gemfile.lock:1]: hmmm...

**Review by bgruening — APPROVED** [review]

**Pull request merged**